CHANGELOG.md

```diff
@@ -67,7 +67,10 @@ The 0.15.0 release has basic support for LLVM 14 but not all features work yet,
 #### Tools
 - Update biosnoop.bt for kernel >=5.17
   - [#2207](https://github.com/iovisor/bpftrace/pull/2207)
+- Add sslsnoop and ssllatency tools
+  - [#2117](https://github.com/iovisor/bpftrace/pull/2117)
```

Comment:
argh sorry for the extra work but can you move this entry up? We did the 0.15 release and this needs to go into `unreleased`

Reply:
done